tests/std/tests/P1135R6_latch/test.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <latch>
+#include <thread>
+
+using namespace std::chrono_literals;

```

Comment:
Should include `<chrono>`.

Reply:
No, latch test don't need timing assumptions

Reply:
Removed `using namespace std::chrono_literals;`